### DISCUSSION TITLE
storage/quickstart.cc failure

### DISCUSSION THREAD

**Discussion by wkyu2kg** [googleapis/google-cloud-cpp#7333] [Q&A] [answered]
It seems that the quickstart.cc program fails due to an error on the WriteObject() function. This happens to v1.28.0~v1.31.1

# ./quickstart gs://useast01
Error creating object: Permanent error in CreateResumableSession: Not Found [NOT_FOUND]

**Comment by coryan** [comment, marked as answer]
Answered in #7334
